File: Phase1/Session-10/complevelinter/src/app/modals/Product.ts
export class Product{
    pid:number;
    name:string;
    price:number;
    rating:number;

    constructor(pid:number,name:string,price:number,rating:number){
        this.pid=pid;
        this.name=name;
        this.price=price;
        this.rating=rating;
    }

    showProducts(){
        console.log(this.pid+" "+this.name+" "+this.price);
        
    }
}

File: Phase2/Session2/src/com/cisco/session2/encaptulation/Product.java
package com.cisco.session2.encaptulation;

public class Product {

	private int id;
	private String name;
	private String desc;
	private double price;
	private double rating;
	
	//get method always returns a value of the same datatype of that Variable
	public int getId() {
		return id;
	}
	//Setter method always void and accept one data to set value
	public void setId(int id) {
		this.id=id;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public String getDesc() {
		return desc;
	}
	public void setDesc(String desc) {
		this.desc = desc;
	}
	public double getPrice() {
		return price;
	}
	public void setPrice(double price) {
		this.price = price;
	}
	public double getRating() {
		return rating;
	}
	public void setRating(double rating) {
		this.rating = rating;
	}
	
	
}
